lib/features/children/add_member/pages/add_family_counter_page.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:givt_app/core/enums/amplitude_events.dart';
+import 'package:givt_app/features/children/add_member/pages/family_member_form_page.dart';
+import 'package:givt_app/features/children/add_member/widgets/smiley_counter.dart';
+import 'package:givt_app/features/family/extensions/extensions.dart';
+import 'package:givt_app/features/family/shared/design/components/components.dart';
+import 'package:givt_app/features/family/shared/widgets/texts/shared_texts.dart';
+import 'package:givt_app/shared/models/analytics_event.dart';
+import 'package:givt_app/shared/widgets/fun_scaffold.dart';
+
+class AddFamilyCounterPage extends StatefulWidget {
+  const AddFamilyCounterPage({super.key});
+  @override
+  State<AddFamilyCounterPage> createState() => _AddFamilyCounterPageState();
+}
+
+class _AddFamilyCounterPageState extends State<AddFamilyCounterPage> {
+  late int _amount;
+
+  @override
+  void initState() {
+    super.initState();
+    _amount = 2;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return FunScaffold(
+      canPop: false,
+      appBar: FunTopAppBar.primary99(
+        title: 'Set up Family',
+      ),
+      body: Column(
+        children: [
+          const TitleMediumText(
+            'How many people are in your family?',
+            textAlign: TextAlign.center,
+          ),
+          const Spacer(),
+          SmileyCounter(totalCount: _amount - 1),
```

Comment:
Maybe rename it because we don't have smileys anymore :'(

Reply:
😭 true